extensionsGallery.json

```diff
@@ -3074,7 +3074,47 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+                {
+                  "extensionId": "62",
+                  "extensionName": "qpi",
+                  "displayName": "Query Performance Insight",
+                  "shortDescription": "ADS UI for the latest version of QPI library",
+                  "publisher": {
+                    "displayName":"Jovan Popovic",
+                    "publisherId": "jocapc",
+                    "publisherName":"jocapc"
+                  },
+                  "versions": [
+                    {
+                      "version": "0.1.6",
+                      "lastUpdated": "03/08/2020",
+                      "assetUri": "",
+                      "fallbackAssetUri": "fallbackAssetUri",
+                      "files": [
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+                          "source": "https://github.com/JocaPC/AzureDataStudio-QPI/releases/download/stable/qpi-0.1.6.vsix"
```

Comment:
Direct VSIX installs should be uploaded to our blob storage account. I've done that - the URL to use is `https://sqlopsextensions.blob.core.windows.net/extensions/qpi/qpi-0.1.6.vsix`

Reply:
Updated